bin/objectFileInfor.rb

```diff
@@ -7,6 +7,10 @@
 # NOTE: I ususally output this script like so: ruby /tmp/rmetz/objectFileInfo.rb /tmp/rmetz/sdrget-2078.txt > /tmp/rmetz/sdrget-2078.csv
 #and then download it for use in excel
 
+# ******************
+# 2019-09-17:  THIS SCRIPT IS DEPRECATED - it has been replaced by the "Checksum Report" bulk action in Argo
+# ******************
+
```

Comment:
how would you feel about printing this upon invocation, in case the person calling has used it before, and doesn't bother looking into the file for usage info?

Reply:
The problem is the REPORT is what is printed.  So we can't.  I wanted to.

Reply:
fair point.  hokey attempt to signal more clearly:  https://github.com/sul-dlss/sdr-services-app/pull/88